packages/advanced-logic/specs/payment-network-any-declarative-0.1.0.md

```diff
@@ -45,10 +47,11 @@ As a payment network, this extension allows to deduce a payment `balance` for th
 | --------------------------------- | ------ | ------------------------------------ | ------------- |
 | **id**                            | String | constant value: "pn-any-declarative" | **Mandatory** |
 | **type**                          | String | constant value: "paymentNetwork"     | **Mandatory** |
-| **version**                       | String | constant value: "0.1.0"              | **Mandatory** |
+| **version**                       | String | constant value: "0.2.0"              | **Mandatory** |
 | **parameters**                    | Object |                                      |               |
 | **parameters.paymentInstruction** | String | Instruction to make payments         | Optional      |
 | **parameters.refundInstruction**  | String | Instruction to make refunds          | Optional      |
+| **parameters.thirdparty**    | [Identity](https://github.com/RequestNetwork/requestNetwork/blob/master/packages/request-logic/specs/request-logic-specification.md#identity-role-and-signature) | Identity of the third party | Optional      |
```

Comment:
use relative link

Reply:
shouldn't the anchor be `#identity` instead?

Reply:
Well, this anchor is giving a better picture of what is an identity.